### PR TITLE
docs updated

### PR DESCRIPTION
**Pull request by anupambhatnagar**
- Added more color to the windows installation section
- Added a Python Version Requirement section

### CODE REVIEW

File: docs/Using-Virtual-Environment.md [hunk 1]
```diff
@@ -45,9 +47,16 @@ the latest pip version using `pip3 install --upgrade pip`
...
 1. To activate the environment execute `$ python-envs\sample-env\Scripts\activate`
 1. Verify pip version is the same as in the __Installing Pip__ section. In case it is not the latest, upgrade to
-the latest pip version using `pip3 install --upgrade pip`
-1. Install ML-Agents package using `$ pip3 install mlagents`
+the latest pip version using `pip install --upgrade pip`
+1. Install ML-Agents package using `$ pip install mlagents`
 1. To deactivate the environment execute `$ deactivate`
+
+Note: 
+- Make sure that the `python` command in the terminal launches a Python3 interpreter. To verify
```
Comment (chriselion) [inline docs/Using-Virtual-Environment.md]: Why not `python --version`?

File: docs/Using-Virtual-Environment.md [hunk 2]
```diff
@@ -45,9 +47,16 @@ the latest pip version using `pip3 install --upgrade pip`
 ## Windows Setup
 
 1. Create a folder where the virtual environments will reside `$ md python-envs`
-1. To create a new environment named `sample-env` execute `$ python3 -m venv python-envs\sample-env`
+1. To create a new environment named `sample-env` execute `$ python -m venv python-envs\sample-env`
 1. To activate the environment execute `$ python-envs\sample-env\Scripts\activate`
 1. Verify pip version is the same as in the __Installing Pip__ section. In case it is not the latest, upgrade to
-the latest pip version using `pip3 install --upgrade pip`
-1. Install ML-Agents package using `$ pip3 install mlagents`
+the latest pip version using `pip install --upgrade pip`
```
Comment (chriselion) [inline docs/Using-Virtual-Environment.md]: The commands are inconsistent - some start with `$` and others don't. I'd lean towards not using them, especially on the windows instructions.

Comment (anupambhatnagar) [reply docs/Using-Virtual-Environment.md]: Ok, I've removed `$` from the windows instructions.

File: docs/Using-Virtual-Environment.md (hunk 1 above)
Comment (anupambhatnagar) [reply docs/Using-Virtual-Environment.md]: Sure, that's better. changed it.

**Comment by chriselion** [comment]
Looks good

**Comment by unityjeffrey** [comment]
hey @anupambhatnagar - i didn't have a chance to review this before it was merged.  i have some suggestions, but it might be faster if I went in and made them myself.  I will do that and send a PR back out.

**Pull request merged**